lib/Options.pm

```diff
@@ -92,7 +94,9 @@ my %cap_defaults = (
 			   quiet => 0,
 			   days => 5,
 			   offset => 0,
+			   timeout => 240,			   
 			   output => undef,
+			   agent => 'Mozilla/5.0 (Linux) XmlTv',			   
```

Comment:
@ejonesnospam may I suggest changing the UA string to simply be `Mozilla/5.0 XMLTV` since it may be used on other platforms than Linux?

According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) the format of the User-Agent header is: `User-Agent: <product> / <product-version> <comment>` where the `<comment>` is optional.